src/browser/input.js

```diff
@@ -155,32 +155,36 @@ const readSelect = compose
       Editable.Select(readSelection(target))
   );
 
-const inputWidth = '460px';
-const inputHeight = '40px';
+const inputWidth = 460;
+const inputHeight = 40;
+const inputXPadding = 32;
 
 const style = StyleSheet.create({
   combobox: {
-    background: '#EBEEF2',
-    borderRadius: '5px',
     height: inputHeight,
     left: '50%',
-    marginLeft: `calc(-1 * (${inputWidth} / 2))`,
+    marginLeft: `calc(-1 * (${inputWidth}px / 2))`,
     position: 'absolute',
-    padding: '0 32px',
     top: '40px',
-    width: `calc(${inputWidth} - ${32 * 2}px)`
+    width: `${inputWidth}px`
   },
   field: {
-    background: 'transparent',
-    borderWidth: 0,
+    background: '#EBEEF2',
+    borderRadius: '5px',
+    borderWidth: '3px',
+    borderStyle: 'solid',
+    borderColor: 'transparent',
     display: 'block',
     fontSize: '14px',
     MozAppearance: 'none',
-    height: inputHeight,
-    lineHeight: inputHeight,
+    height: `${inputHeight - 6}px`,
+    lineHeight: `${inputHeight - 6}px`,
     margin: 0,
-    padding: 0,
-    width: `calc(${inputWidth} - ${32 * 2}px)`
+    padding: `0 ${inputXPadding}px`,
+    width: `calc(${inputWidth - 6}px - ${inputXPadding * 2}px)`
```

Comment:
Oops, you are right. This is a mistake left over from the previous implementation.